content/changelog/2024-10-11.md

```diff
@@ -0,0 +1,59 @@
+
+![10 projects on the Neon Free Plan](/docs/relnotes/ten_projects.png)
+
+You may have seen the announcement already &#8212; **Neon Free Plan users can now create up to 10 projects**. We hope this increase will make it easier for you to learn a new stack, build the AI-powered app you've been thinking about, ship an MVP, upgrade to the next version of Postgres, or anything else the previous limit kept you from doing. **Why did we make this change?** Read all about it here: [10x-ing our Free Plan: Everyone Gets Ten Projects](https://neon.tech/blog/10x-projects-on-free-plan).
+
+To create a new project, navigate to the [Projects](https://console.neon.tech/app/projects) page in the Neon Console and click **New Project**.
+
+_Please note that [Free Plan account-level allowances](https://neon.tech/docs/introduction/plans#free-plan) for compute hours, storage, and bandwidth remain unchanged._
+
+### Drag-to-zoom monitoring graphs
+
+We've enhanced the graphs on our **Monitoring** page to support drag-to-zoom. You can now click and drag to zoom in on a selected range.
+
+![drag-to-zoom monitoring graphs](/docs/relnotes/drag-to-zoom.png)
+
+### Don't forget to grab your ticket for Neon Deploy
+
+Join us on **October 30th** at **10:00 AM** PT to learn how Neon helps developers ship faster with Postgres. [Get your ticket now!](https://neon.tech/deploy)
+
+<details>
+<summary>**Fixes & improvements**</summary>
+
+- **Neon CLI update**: We released a new version of the [Neon CLI](https://neon.tech/docs/reference/neon-cli), with the following updates:
+
+  - Fixed an issue with the `login` alias for the [neon auth](/docs/reference/cli-auth) command.
+  - Removed the previously deprecated `set-primary` primary subcommand from the [branches](/docs/reference/cli-branches) command. It was replaced by the [set-default](/docs/reference/cli-branches#set-default) subcommand.
+  - Removed the previously deprecated `--allow-list` and `--ip-primary-only` options from the [projects update](/docs/reference/cli-projects#update) command. Operations performed by these options are supported by the [ip-allow](/docs/reference/cli-ip-allow) command.
+  - Removed the previously deprecated `--primary-only` option from the [ip-allow](/docs/reference/cli-ip-allow) command. It was replaced by the [ip-allow add --protected-only](https://neon.tech/docs/reference/cli-ip-allow#add) option.
+  - Updated the [project list](/docs/reference/cli-projects#list) command to output a user-friendly message when there are no projects or shared projects to display.
+
+  To update your Neon CLI installation to the latest version, follow our [CLI upgrade instructions](/docs/reference/cli-install#upgrade).
+
+- **Create Support Ticket modal enhancements**: We've made a few updates to the **Create support ticket** modal in the Neon Console:
+
+  - Added a drop-down menu for selecting a personal account or organization.
+  - Added support for selecting a shared project.
```

Comment:
I think support tickets for shared project was available already.

Reply:
Thank you. I'll remove it.